tee-worker/omni-executor/native-task-handler/src/lib.rs

```diff
@@ -150,7 +150,8 @@ pub async fn run_native_task_handler<
 
 	tokio::spawn(async move {
 		while let Some((wrapper, sender)) = receiver.recv().await {
-			handle_native_task(ctx.clone(), wrapper, sender).await;
+			let ctx_cloned = ctx.clone();
+			tokio::spawn(async move { handle_native_task(ctx_cloned, wrapper, sender).await });
```

Comment:
We need some kind of backpressure here as tasks could get spawned indefinitely and cause memory overhead 
Even though the receiver channel is bounded, it doesn't enforce real backpressure since messages are consumed quickly and tasks are still spawned indefinitely.

Reply:
Thank you - that's a good point

I use semaphore to limit it. In the long run I think it makes sense to not send the tasks to native-task-handler at all, as jsonrpsee should be handle async registered rpc concurrently already

Reply:
Yeah we should try to move away from the native-task-handler this pattern was used because of our limitations with `sgx` dependencies and to isolate from block productions.